h2o-assembly/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
   if (project.hasProperty("doIncludeOrc") && project.doIncludeOrc == "true") {
     compile project(":h2o-orc-parser")
   }
+  compile project(":h2o-parquet-parser")
```

Comment:
Yes, it will include whatever is the defined to be used in the H2O assembly (it won't override any setting).